src/js/utils/index.ts

```diff
@@ -59,12 +100,9 @@ function vTriggerMerge(rawFrameData, vtState) {
 }
 
 // this allow me to build the JSON for the setPlayer action creator in selectCharacter, SegmentSwitcher and ____ componenet
-export function helpCreateFrameDataJSON(rawFrameData, moveNameType, inputNotationType, normalNotationType, vtState) {
+export function helpCreateFrameDataJSON(rawFrameData, dataDisplayState: DataDisplaySettingsReducerState, vtState: VtState) {
   
-  const dataToRename = vtState === "normal"
-    ? rawFrameData.normal
-    : vTriggerMerge(rawFrameData, vtState);
-
-  return moveNameType === "official" ? dataToRename : renameData(dataToRename, moveNameType, inputNotationType);
+  const dataToRename = (vtState === "normal") ? rawFrameData.normal : vTriggerMerge(rawFrameData, vtState);
 
+  return (dataDisplayState.moveNameType === "official") ? dataToRename : renameData(dataToRename, dataDisplayState);
```

Comment:
Was looking at this again and realized that we probably _don't_ need to have this ternary operator anymore since this was originally handling the decision making for whether or not to rename data. The decision making is within the `renameData` function itself now, so we might actually want to change this to `return renameData(dataToRename, dataDisplayState)`.

I'm testing this now as I'm applying the changes requested and it's working as expected.